package.json

```diff
@@ -54,7 +54,7 @@
     "semver": "^5.4.1",
     "sequelize": "^4.11.1",
     "sequelize-typescript": "^0.5.0",
-    "sqlite3": "^3.1.12",
+    "sqlite3": "^4.0.3",
```

Comment:
Looking at [the changelog](https://github.com/mapbox/node-sqlite3/blob/master/CHANGELOG.md): my problems were probably caused by either the old node-pre-gyp version bundled with 3.1.13 or the lack of Node 10 support before 4.0.1. The bump from v3 to v4 seem to be nothing but the removal of support for old Node versions.

Given that Node.js 10 will be the latest LTS starting on tuesday I think supporting it is a good goal.

> probably more than it should because sqlite is only used in development

Given that sqlite is the example config and isn't cautioned against anywhere I would be willing to bet that the majority of Nucleus instances outside Atlassian do use sqlite in production.